src/js/base/model.js

```diff
@@ -98,7 +105,22 @@ export default Backbone.Model.extend(extend({
 
     return isFunction(messages[category]) ? messages[category] : this[messages[category]];
   },
-  handleMessage({ category, resource, author, payload }) {
+  queueMessage(data) {
+    if (!this.notifications) {
+      this.notifications = [data];
+      return;
+    }
+
+    this.notifications.push(data);
+  },
+  handleMessage(data) {
+    if (this.isFetching()) {
```

Comment:
I think extending the attributes with the updated_at should happen here so that it's stored with the queue.. that timestamp is when the action occurred, not when it was processed.